runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -631,12 +615,26 @@ public void onFocusChanged(FocusChanged focusChanged)
 
 	private void notifyHighlightedItem(GroundItem item)
 	{
-		boolean shouldNotifyHighlighted = config.highlightedColor().equals(getHighlighted(
+		final Color highlightColor = getHighlighted(
 			new NamedQuantity(item),
 			item.getGePrice(),
-			item.getHaPrice()));
+			item.getHaPrice());
+
+		boolean shouldNotifyHighlighted = config.highlightedColor().equals(highlightColor);
 
-		if (!config.notifyHighlightedDrops() || !shouldNotifyHighlighted)
+		boolean shouldNotifyTier = getValueByMode(item.getGePrice(), item.getHaPrice()) > config.notifyTier().getValueFromTier(config) &&
+			FALSE.equals(hiddenItems.getUnchecked(new NamedQuantity(item)));
+
+		String dropType;
+		if (config.notifyHighlightedDrops() && shouldNotifyHighlighted)
+		{
+			dropType = "highlighted";
+		}
+		else if (config.notifyTier() != HighlightTier.OFF && shouldNotifyTier)
```

Comment:
Isn't `config.notifyTier() == HighlightTier.OFF` already implicitly covered by the `getValueFromTier()` method call in `shouldNotifyTier`?